apps/ledger-live-desktop/src/renderer/families/polkadot/NominationFlowModal/steps/StepConfirmation.tsx

```diff
@@ -23,7 +24,35 @@ const Container = styled(Box).attrs(() => ({
 }>`
   justify-content: ${p => (p.shouldSpace ? "space-between" : "center")};
 `;
-function StepConfirmation({ t, optimisticOperation, error, signed }: StepProps) {
+function StepConfirmation({
+  t,
+  optimisticOperation,
+  error,
+  signed,
+  transaction,
+  source,
+}: StepProps) {
+  const preloaded = usePolkadotPreloadData();
+  const { validators: allValidators } = preloaded;
+
+  const validators = useMemo(() => {
+    return allValidators
+      .filter(val => transaction.validators.includes(val.address))
+      .map(val => val.identity || val.address);
+  }, [allValidators, transaction.validators]);
+
+  useEffect(() => {
+    if (optimisticOperation && validators) {
+      track("staking_completed", {
+        currency: "DOT",
+        validator: validators,
```

Comment:
[Question] Do we not want 1 validator to be returned here and not an array of validators?

Reply:
I guess not, DOT support a list of validators. Do not ask me why...

Reply:
Ah ok, good to know for the future. Thanks.